src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCtorCall.java

```diff
@@ -79,13 +79,15 @@ public Valid() { //indent:4 exp:4
     } //indent:4 exp:4
 
     public Valid(InputIndentationCtorCall obj, long arg) { //indent:4 exp:4
-      obj.super( //indent:6 exp:6
-          arg); //indent:10 exp:10
+      obj //indent:6 exp:6
+          .super( //indent:10 exp:10
+            arg); //indent:12 exp:12
```

Comment:
how come this wants 12 (+2) but other modified test case wants 14 (+4)?

Reply:
Both +2 and +4 are acceptable. The config is
 * basicOffset = 2
 * lineWrappingIndentation = 4